src/services/expense.service.js

```diff
@@ -0,0 +1,75 @@
+'use strict';
+
+const { getNewId } = require('../utils/getNewId.js');
+
+let expenses = [];
+
+function resetExpenses() {
+  expenses = [];
+};
+
+const getAll = () => expenses;
```

Comment:
```suggestion
const resetExpenses = () => {
  expenses = [];
};

const getAll = () => expenses;
```
Use the same style in your codebase

Reply:
I understud 🤝